distribution/docs/src/main/jdocs/templates/documentation.ftl

```diff
@@ -0,0 +1,31 @@
+= ${branding-expanded} Documentation
+include::${project.build.directory}/doc-contents/_contents/jconfig.adoc[]
+
+
+:sectnums!:
+== Introduction
+:sectnums:
+
+include::{adoc-include}/_introduction/introduction-contents.adoc[]
```

Comment:
This is a file that is unlikely to be renamed/split further and is context-dependent regarding placement, so it is hard-coded/fixed.